pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -206,3 +255,109 @@ func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Networ
 	c.recorder.Eventf(network, corev1.EventTypeNormal, "processNetworkCreation", "successfully created network from mizar cluster: %v.", context)
 	return nil
 }
+
+func createVpcOrSubnetCRD(tenant, vpcOrSubnetName, defaultTemplatePath string, discoveryClient discovery.DiscoveryInterface, dynamicClient dynamic.Interface) error {
```

Comment:
is this func creating both vpc and subnet, or either one?

Reply:
this function is creating either one via different name defined in parameter 'vpcOrSubnetName'.